### ISSUE TITLE
Toolbar Outlook 2007 hides only temporarly

### ISSUE THREAD

**Issue by Offerel** [aluxnimm/outlookcaldavsynchronizer#102]
If i hide or disable the Toolbar (not the Plugin) in Outlook 2007, it comes back after restarting Outlook 2007. If i disable or hide other Toolbars, they stay hidden/disabled also after restarting Outlook 2007.

**Comment by aluxnimm** [comment]
PierreMarieBaty implemented the OL2007 support. see #65 
we don't have a test machine at,  maybe he can check it.

**Comment by PierreMarieBaty** [comment]
See my answer at #65

**Comment by Offerel** [comment]
I have made a Workaround for me. I have added a small VBA Script in Application Startup of ThisOutlookSession with the following Code:

`Application.ActiveExplorer.CommandBars("CalDav Synchronizer").Visible = False`
With this, i can show the Commandbar manually if i want and it hides with the next start of Outlook (or if i hide it manually).

**Comment by aluxnimm** [comment]
Thanks for the info, if I have some time I will check the proposed solution to save the info in the registry,

**Comment by aluxnimm** [comment]
Save Outlook 2007 toolBar position and visibility in Registry, implemented in 2.3.0

**Comment by aluxnimm** [comment]
@PierreMarieBaty  can you  check if  the new code saves the toolbar position  correctly please?

**Comment by PierreMarieBaty** [comment]
@aluxnimm it does not exactly. The toolbar now appears vertical, clamped to the left border of the Outlook window, and when I drag it to a horizontal position this state is not saved when I restart Outlook, i.e. it still appears vertical. But note that the toolbar offset seems somewhat saved, at the exception that the horizontal offset to which I placed it has become a vertical offset, although clamped by the max height of the Outlook window.

I must add that I no longer actively develop on Microsoft systems. The stubbornly repeated dead-wrong choices of Microsoft regarding their main operating system and their understanding of privacy, and the enthusiastic reaction of the cattle, made me finally switch to a slightly lesser evil, i.e. Mac. In consequence I no longer have a usable machine for Win32 development. I nevertheless have to say that I did enjoy Outlook CalDAV Synchronizer while I used it. Congratulations for having created such a needed tool.
